core/ui_mixins/quick_panel.py

```diff
@@ -140,7 +148,9 @@ def show_paginated_panel(items, on_done, flags=None, selected_index=None, on_hig
             selected_index=selected_index,
             on_highlight=on_highlight,
             limit=limit,
-            next_message=next_message)
```

Comment:
👍 It sounds a better refactoring. I always have troubles keeping the values from being `None`.